source/api/rdf-dcat.rst

```diff
@@ -2,3 +2,21 @@
 
 RDF DCAT end point
 ##################
+
+IN PROGRESS
+
+The RDF DCAT end point provides a way of getting information about the catalog, the datasets and services, and links to distributed resources in a machine-readable format. The formats of the output are based on DCAT, an RDF vocabulary that is designed to facilitate interoperability between web-based data catalogs.
```

Comment:
We should precise the version of DCAT probably but I'm not even sure of the version we have as first development was made as it was only a W3C recommendation and then we added to CSW maybe with DCAT-AP EU in mind and we have a plugin with 1.2. And version 2 is coming. So probably not that easy to answer :(